### ISSUE TITLE
Add method to check if a Scenario exists

### ISSUE THREAD

**Issue by arcanaxion** [Avaiga/taipy#1100]
**Description**
Took me awhile to realise that I could do `tp.exists(scenario.id)`, because I was looking through the members of Scenario to find how to check if it had been deleted.

Might be helpful if Scenario (and other relevant entities) had a method to check if it has been deleted.

**Acceptance Criteria**
- [ ] Ensure new code is unit tested, and check code coverage is at least 90%
- [ ] Propagate any change on the demos and run all of them to ensure there is no breaking change
- [ ] Ensure any change is well documented

**Comment by trgiangdo** [comment]
Do you mean exposing new methods like:
```python
scenario.exists()
task.exists()
```

**Comment by arcanaxion** [comment]
> Do you mean exposing new methods like:
> 
> ```python
> scenario.exists()
> task.exists()
> ```

Yup, that's what I mean

**Comment by ioannastantzou** [comment]
hello! I am a first time contributor ,do you think I could work on this issue?

**Comment by trgiangdo** [comment]
Yes absolutely. The `tp.exists()` API is already implemented. All we need to do is exposing the `exists()` API in the DataNode, Task, Scenario, ... classes.

Please let us know if you have any question.

**Comment by trgiangdo** [comment]
Hello @ioannastantzou, this issue is currently under discussion by the Taipy team as shown in the label. It was my mistake to assign an under discussion issue to you. I'm really sorry about that.

We will continue the discussion to see whether this new feature should be a part of Taipy package or not. Please be patient with us.

In the meantime, you can find issues with the label "Help wanted" or "good first issue" since those are the features or bugs that we want to include / fix in the Taipy package.

**Comment by ioannastantzou** [comment]
okay no worries! Could you please check out issue #375 , i have asked to be assigned for this one! thank you!
